src/controllers/apiController.ts

```diff
@@ -337,28 +337,37 @@ export class ApiController extends BaseController {
    * header.
    *
    * @param {string} deviceToken Token returned from createParticipant call
+   * @param {string} voiceCallId The callId returned by the Voice platform 
    * @param {string} [sipUri=sip:sipx.webrtc.bandwidth.com:5060] SIP URL to transfer to
    *
    * @returns BXML string
    */
-  static generateTransferBxml(deviceToken: string, sipUri='sip:sipx.webrtc.bandwidth.com:5060'): string {
-    return '<?xml version="1.0" encoding="UTF-8" ?>\n'
-        + '<Response>'
-        + this.generateTransferBxmlVerb(deviceToken, sipUri)
-        + '</Response>';
+  static generateTransferBxml(deviceToken: string, voiceCallId: string, sipUri='sip:sipx.webrtc.bandwidth.com:5060'): string {
+      return '<?xml version="1.0" encoding="UTF-8" ?>\n'
+          + '<Response>'
+          + this.generateTransferBxmlVerb(deviceToken, voiceCallId, sipUri)
+          + '</Response>';
   }
 
   /**
    * Generates the BXML verb to transfer a call into a WebRTC session (not wrapped in a Response element).
    *
    * @param {string} deviceToken Token returned from createParticipant call
+   * @param {string} voiceCallId The callId returned by the Voice platform 
    * @param {string} [sipUri=sip:sipx.webrtc.bandwidth.com:5060] SIP URL to transfer to
    *
    * @returns BXML string
    */
-  static generateTransferBxmlVerb(deviceToken: string, sipUri = 'sip:sipx.webrtc.bandwidth.com:5060'): string {
-    return '<Transfer>\n'
-        + `\t<SipUri uui="${deviceToken};encoding=jwt">${sipUri}</SipUri>\n`
-        + '</Transfer>';
+  static generateTransferBxmlVerb(deviceToken: string, voiceCallId: string, sipUri = 'sip:sipx.webrtc.bandwidth.com:5060'): string {
+      if (voiceCallId != undefined) {
```

Comment:
super nit, works as is but I generally like to use triple equals `===` and/or for undefined checks just use the `!`. Don't have to change if you don't wish

Reply:
Fixed to if (voiceCallId) {... }, thank you!